### PR TITLE
Internationalization infra

### PR DESCRIPTION
**Pull request by jniederm**
Please see https://github.com/oVirt/ovirt-web-ui/wiki/Internationalization

### CODE REVIEW

File: zanata.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<config xmlns="http://zanata.org/namespace/config/">
+    <url>https://translate.zanata.org/</url>
+    <project>ovirt-web-ui</project>
+    <project-version>1.1.0</project-version>
```
Comment (mareklibra) [inline zanata.xml]: Recent version is 1.2.0. I would keep these two project versions in sync

File: package.json [hunk 2]
```diff
@@ -44,13 +48,15 @@
...
     "http-proxy-middleware": "0.17.1",
+    "intl-messageformat": "^2.0.0",
     "jest": "19.0.2",
     "json-loader": "0.5.4",
     "object-assign": "4.1.0",
     "opn": "4.0.2",
     "path-exists": "3.0.0",
     "postcss-loader": "0.13.0",
     "promise": "7.1.1",
+    "react-intl": "^2.3.0",
```
Comment (mareklibra) [inline package.json]: Please avoid '^', specify exact version (due to ovirt-nodejs-modules ...)

File: scripts/intl/normalize-messages.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+function main() {
+  const stringContent = fs.readFileSync(SOURCE, { encoding: 'utf8' })
+  const parsedContent = JSON.parse(stringContent)
+  const serializedContent = stableStringify(parsedContent, { space: 2 })
+  fs.writeFileSync(DESTINATION, serializedContent)
+  console.log(chalk.green(`[normalize-messages.js] ${DESTINATION} written ✔`))
+}
+
+main()
```
Comment (mareklibra) [inline scripts/intl/normalize-messages.js]: \n

File: scripts/intl/underscore2dash.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    .filter(item => item.match !== null)
+    .forEach(item => {
+      const oldFile = path.join(DIRECTORY, item.original)
+      const newFile = path.join(DIRECTORY, `${item.match[1]}-${item.match[2]}`)
+      fs.renameSync(oldFile, newFile)
+      console.log(chalk.green(`[underscore2dash.js] ${oldFile} -> ${newFile} ✔`))
+    })
+}
+
+main()
```
Comment (mareklibra) [inline scripts/intl/underscore2dash.js]: \n

File: src/intl/index.test.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+import translatedMessages from './translated-messages.json'
+import { localeDataMap } from './index'
+
+describe('intl', () => {
+  it('should import all necessary local data', () => {
+    [...Object.keys(translatedMessages), 'en']
+      .forEach(availableLocale => expect(Object.keys(localeDataMap)).toContain(availableLocale))
+  })
+})
```
Comment (bond95) [inline src/intl/index.test.js]: \n

File: scripts/intl/normalize-messages.js (hunk 3 above)
Comment (jniederm) [inline scripts/intl/normalize-messages.js]: done

File: scripts/intl/underscore2dash.js (hunk 4 above)
Comment (jniederm) [inline scripts/intl/underscore2dash.js]: done

File: src/intl/index.test.js (hunk 5 above)
Comment (jniederm) [inline src/intl/index.test.js]: done

File: package.json [hunk 6]
```diff
@@ -31,6 +34,7 @@
     "eslint-config-standard-react": "~4.2.0",
     "eslint-loader": "1.5.0",
     "eslint-plugin-flowtype": "2.32.1",
+    "eslint-plugin-import": "^2.7.0",
```
Comment (mareklibra) [inline package.json]: Remove '^' , please

File: scripts/intl/extract-messages.js [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    }
+    return messageDescriptor
+  }
+}
+
+function main() {
+  const reactIntlMessages = normalizeMessages(messages)
+  mkdirpSync(OUTPUT_DIR)
+  fs.writeFileSync(OUTPUT_FILE, JSON.stringify(reactIntlMessages, null, 4))
+  console.log(chalk.green(`[extract-messages.js] ${OUTPUT_FILE} written ✔️`))
```
Comment (mareklibra) [inline scripts/intl/extract-messages.js]: "✔️" :-)

File: package.json (hunk 6 above)
Comment (jniederm) [inline package.json]: done

**Pull request merged**